influxql/engine.go

```diff
@@ -408,7 +409,8 @@ func (m *MapReduceJob) processRawQueryDerivative(lastValueFromPreviousChunk *raw
 
 		// Calculate the derivative of successive points by dividing the difference
 		// of each value by the elapsed time normalized to the interval
-		diff := v.Values.(float64) - lastValueFromPreviousChunk.Values.(float64)
+		diff := i64tof64(v.Values) - i64tof64(lastValueFromPreviousChunk.Values)
```

Comment:
It seems to me to be more idiomatic to spell this out. `int64toFloat64`. Look at the math package.